cves/CVE-2018-1336.yml

```diff
@@ -103,9 +106,9 @@ unit_tested:
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
     Must be just "true" or "false".
-  answer:
-  code:
-  fix:
+  answer: The  unit tests were improved to check for supplementary characters.
```

Comment:
Could you state whether there were automated unit tests for the subsystem identified prior to the fix?

Reply:
There were unit tests written but they did not check for this vulnerability.